GOF/app/views/characters/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<head><title>Adding Character...</title></head>
+
+<% @character.errors.full_messages.each do |message| %>
+<p><%= message %></p>
+<% end %>
+
+
+<h1> Add Character </h1>
+<%= form_for (@character) do |t| %>
+<%= t.label :Name %>
+<%= t.text_field :name %>
+
+<%= t.label :Age %>
+<%= t.text_field :age %>
+
+<%= t.label :Gender %>
+<%= t.text_field :gender %>
+
+<%= t.label :Image %>
+<%= t.text_field :img_url %>
+
+<%= t.label :House_Number %>
+<%= t.text_field :house_id %>
+
+<%= t.submit %>
+<% end %>
```

Comment:
the convention for for forms is to use f instead of t -- t is used in migrations. they also look similar, so it could potentially be confusing if you are jumping around between the files of the rails app. also ident!

all that being said 👍 👍 👍